config/canonical.json

```
@@ -5816,7 +5816,9 @@
     "tags": {
       "amenity": "cafe",
       "brand": "Segafredo",
-      "name": "Segafredo"
+      "name": "Segafredo",
+      "brand:wikidata": "Q21282762",
+      "brand:wikipedia": "en:Massimo_Zanetti_Beverage_Group"
```

Can you remove the `_` underscores?  thanks!
```suggestion
      "brand:wikipedia": "en:Massimo Zanetti Beverage Group"
```